developmentseed/os-climate-hazard
# Separate source data information from model information

## What I am changing

The hazard indicator (model) descriptions currently include information about the source data, limited to CMIP6 and the default mode, scenario, and year selections.

Now that the source data is variable (flexible subset of CMIP6 models, alternative data source UKCP18), the hard-coded data source information may not be valid.

This PR strips the **source data information** from the hazard indicator descriptions and moves it to a separate file. It also adds a description for the UKCP18 source.

## How I did it

Remove the last paragraph from all the indicator descriptions, copied a sentence on CMIP6 to a new file in `/sources`.

## How to test it

Run a CMIP6 and a UKCP18 based workflow and verify that the revised model information ends up in the output.

## Possible follow-up

With this change, indicator descriptions returned by the API will include neither inaccurate nor accurate information about the source data. We could change that, such that this information becomes available to users.
